metricflow/dataflow/dataflow_plan.py

```diff
@@ -673,8 +739,10 @@ def __init__(  # noqa: D
         self,
         parent_node: BaseOutput[SourceDataSetT],
         include_specs: Sequence[InstanceSpec],
+        hide_description: bool = False,
```

Comment:
Instead of hiding the description, what about a different description that better describes the operation You were looking to filter out measures that have different additive properties?

Reply:
replaced with `replace_description` where we would pass a string to replace the current description! For example, here when we filter only semi additive measures it would say,
```
<FilterElementsNode>
                <!-- description = Filter all Semi-additive Measures -->
                <!-- node_id = pfe_6 -->
```